src/database/index.ts

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FirebaseApp } from '../firebase-app';
+import * as firebaseAdmin from '../index';
+import * as firebaseRtdbApi from '@firebase/database';
+import * as firebaseRtdbTypesApi from '@firebase/database-types';
```

Comment:
Ok, as per the recommendation from the JS SDK team, let's use `@firebase/database` for everything. Let's assume their implementation will sync up with the typings eventually (refer to the issues you've reported).

Reply:
Commenting for visibility, we're switching back now to `@firebase/database-types`.